src/page/AddMenu/components/MenuPrice/index.tsx

```diff
@@ -24,34 +61,91 @@ export default function MenuPrice({ isComplete }:MenuPriceProps) {
               <div className={styles.mobile__header}>
                 <div className={styles['mobile__header-title--complete']}>가격</div>
               </div>
-              <div className={styles['mobile__price-info-text-box']}>
-                <div className={styles['mobile__price-info-text']}>
-                  <div className={styles['mobile__price-info-text__price']}>
-                    {singlePrice}
-                    원
+              {isSingle ? (
+                <div className={styles['mobile__price-info-text-box']}>
+                  <div className={styles['mobile__price-info-text']}>
+                    <div className={styles['mobile__price-info-text__price']}>
+                      {singlePrice}
+                      원
+                    </div>
                   </div>
                 </div>
-              </div>
+              )
+                : (optionPrices || []).map((input) => (
+                  <div key={input.id} className={styles['mobile__price-info-text-box']}>
+                    <div className={styles['mobile__price-info-text']}>
+                      <div className={styles['mobile__price-info-text__size']}>
+                        {input.option}
+                      </div>
+                      <div>
+                        /
+                      </div>
+                      <div className={styles['mobile__price-info-text__price']}>
+                        {input.price}
+                        원
+                      </div>
+                    </div>
+                  </div>
+                ))}
             </>
           ) : (
             <>
               <div className={styles.mobile__header}>
                 <div className={styles['mobile__header-title']}>가격</div>
               </div>
-              <div className={styles['mobile__price-info-input-box']}>
-                <div className={styles['mobile__price-info-inputs']}>
-
-                  <div className={styles['mobile__price-info-inputs__price-input-box']}>
-                    <input
-                      type="number"
-                      className={styles['mobile__price-info-inputs__price-input']}
-                      value={singlePrice}
-                      onChange={(e) => setSinglePrice(Number(e.target.value))}
-                    />
-                    <p className={styles['mobile__price-info-inputs__price-input-won']}>원</p>
+              {isSingle
+                ? (
+                  <div className={styles['mobile__price-info-input-box']}>
+                    <div className={styles['mobile__price-info-inputs']}>
+                      <div className={styles['mobile__price-info-inputs__price-input-box']}>
+                        <input
+                          type="number"
+                          className={styles['mobile__price-info-inputs__price-input']}
+                          value={singlePrice === 0 || singlePrice === null ? '' : singlePrice}
```

Comment:
사용자가 0원을 입력할 수 도 있으니 undefined로 놓는 거 어떨까요? 하고 number 타입쪽에 inputMode 추가해 주면 좋을 것 같아요

```suggestion
                          inputMode="decimal"
                          value={singlePrice === 0 || singlePrice === null ? undefined : singlePrice}
```

Reply:
imputMode를 이용해서 모바일에서의 편의성을 생각하지 못했네요 바로 수정하겠습니다.

Reply:
0은 초기값이 0이라서 0이 보이는것을 방지하기 위해서 이렇게 작성하게되었습니다.